Makefile

```diff
@@ -1,229 +1,147 @@
-########################################################################################################
-#@author: Duy DINH
-#@date: Oct. 2022
-#@pre-conditions: 
-# We suppose that you already clone the model2owl using github command line below
-# $ git clone https://github.com/OP-TED/model2owl.git
-#
-# After cloning into your local repository
-# $ cd model2owl
-#
-# To see the content of the Makefile, please use the command below
-# $ less Makefile
-#
-########################################################################################################
-#curBranch := $(shell git branch --show-current)
-curBranch := $(shell git rev-parse --abbrev-ref HEAD)
-# Number of file under .git repo (to test if the git repo was already cloned)
-gitRepoFiles :=  $(shell ls -l transform/model2owl/.git >/dev/null 2>&1 | wc -l )
-secrets.GITHUB_TOKEN := *******
-
 # Model2owl directory
-MODEL2OWL_DIR?="."
-# Project directory, ie. directory where we store the input/output from model2owl
-PROJECT_DIR?=".."
-# One of the xmi input files to be merged into a single UML xmi/xml file
-FIRST_INPUT_XMI_FILE?="test/test-multi-xmi/ePO_CM.xml"
-# Input dir containing files to be merged
-INPUT_XMI_DIR?=$(shell dirname ${FIRST_INPUT_XMI_FILE})
+MODEL2OWL_DIR?=.
 # rdflib version
 RDF_LIB_VERSION?=6.2.0
-SAXON?="../saxon-he-10.6.jar"
-# Output directory containing combined file from multiple xmi / xml UML models
-COMBINED_XMI_DIRECTORY?="output/combined-xmi"
-COMBINED_FILE_NAME?=ePO-combined.xmi
-XMI_DIRECTORY?="../output/combined-xmi"
+#Saxon path
+SAXON?=${MODEL2OWL_DIR}/saxon/saxon-he-10.6.jar
```

Comment:
version is hardcoded here, and it is hardcoded below in the target. Please unify, move the reference to a variable

Reply:
or, have a standard versionless name, which is teh result of the renaming the downloaded jar